src/main/java/ojosama/talkak/category/repository/CategoryRepository.java

```diff
@@ -0,0 +1,10 @@
+package ojosama.talkak.category.repository;
+
+import java.util.Optional;
+import ojosama.talkak.category.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CategoryRepository extends JpaRepository<Category, Long> {
+
+    Optional<Category> findByCategory(String category);
```

Comment:
```suggestion
    Optional<Category> findByName(String name);
```
카테고리를 카테고리로 찾는다는 메서드명이 모호한 느낌이 들어서 `name`이나 다른 이름을 쓰는 게 좋을 것 같아요. 지금 보니 erd에 category로 되어 있네요..저는 변경하는 게 좋을 것 같은데 어떻게 생각하시나요!

Reply:
좋은 것 같습니다 name으로 변경하면 될 것 같아요!